expense/expenseRouter.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const router = express.Router();
+
+const Expense = require('./expenseModel');
+
+// Set Endpoints Here
+router
+    .route('/')
+    .get((req, res) => {
+        Expense
+            .find({})
+            .populate('budget', '_id title budgetAmount' )
```

Comment:
Solid! I like the well placed use of both populates here for budget and category inside the Expense.